data/ca/legislature/Miguel-Santiago-d666349e-e1e1-4109-855e-a5cc1ce87bc6.yml

```diff
@@ -0,0 +1,25 @@
+id: ocd-person/d666349e-e1e1-4109-855e-a5cc1ce87bc6
+name: Miguel Santiago
+given_name: Miguel
+family_name: Santiago
+image: https://www.assembly.ca.gov/sites/assembly.ca.gov/files/memberphotos/AD53_Santiago_Portrait150_20141201.jpg
+party:
+- name: Democratic
+roles:
+- start_date: '2021-05-24'
```

Comment:
@bfossen-ce on 12/05/2022 this person started as rep of the 54th district but from 12/01/2014 until that point he was rep in the 53rd district. Not sure if that needs to be reflected here.